hawc/apps/summary/models.py

```diff
@@ -249,6 +249,8 @@ def clean(self):
             self.get_table()
         except PydanticError as e:
             raise ValidationError({"content": e.json()})
+        except Exception as e:
```

Comment:
Could you change to the specific exception type instead of a catch all?

Reply:
Done!